src/bikeracks/votes.py

```diff
@@ -111,5 +111,53 @@ def submit_vote():
     # maybe return 500 here
     return "", 500
     
-
+@votes.route('/unvote', methods=['POST'])
+def unvote():
+    # remove a vote that the user had previously made
+    if request.method == 'POST':
```

Comment:
Request.method is guaranteed to be `'POST'`. That's the point of the decorator.

If you want to play it safe here then change this to be a negative so you don't have to indent the next 50 lines.

```
if request.method != 'POST':
  return "Bad Request", 400
```

Reply:
👍